### DISCUSSION TITLE
QuickLook

### DISCUSSION THREAD

**Discussion by mbrotz** [beeware/briefcase#1909] [Q&A] [answered]
Hi

I would like to develop a quicklook plugin with an embedded python interpreter. I want to use python to generate the preview using html. 

However, after much googling i found no resources at all regarding this topic. 

Is it even possible with briefcase? And how would i do it?

Thanks very much. 
Manuel

**Comment by freakboy3742** [comment, marked as answer]
If, by "Quicklook", you're referring to a macOS Finder plugin interface - it might be possible to build a plugin as you describe, but I can't provide any real hints on how you'd go about it. It definitely isn't an option that Briefcase (or any other part of BeeWare) provides out of the box.

If you want to try an implement it yourself, what you'll be looking for is a native macOS tutorial written in Objective C; you should then be able to invoke the same APIs from Python using the Rubicon bridge.

**Reply by mbrotz** [reply]
Yes, i was referring to a finder plugin. I know how to make a plugin in obj c and i found some tutorials on embedding a python interpreter into a mac app. So i will try that. I was just wondering if it was possible to do it using briefcase. Would have made stuff simpler. 

Thank you anyway!

Manuel
